hack/create-user-workspace.sh

```diff
@@ -22,12 +22,8 @@ then
   export KUBECONFIG=${KCP_KUBECONFIG}
 fi
 
-echo "Accessing the home workspace:"
-kubectl ws '~'
-
-if [ "${ROOT_WORKSPACE}" == "~" ]; then
-  ROOT_WORKSPACE=$(kubectl ws . --short)
-fi
+echo "Accessing the root workspace:"
+kubectl ws $ROOT_WORKSPACE
```

Comment:
we don't use `flags.sh` (nor preview.sh) in this script (yet). 
you can run create-user-workspace.sh and it should automatically create appstudio workspace in your home workspace and bind everything from `root:`

Reply:
the script is called from preview.sh, so at least for that usecase it will be working, but could break current usecase. Created #778 for now which should be safe.

Reply:
Code updated to use '~' by default but allow to modify using HOME_WORKSPACE